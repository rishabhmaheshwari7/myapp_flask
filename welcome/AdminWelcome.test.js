import { render, screen } from "@testing-library/react"
import AdminWelcome from './AdminWelcome.component'
import { MemoryRouter } from "react-router-dom"
describe('Welcome Admin Component', () => {

    it('Should render the admin welcome text correctly', () => {
        render(<MemoryRouter><AdminWelcome /></MemoryRouter>)
        expect(screen.getAllByRole("welcome")).toMatchSnapshot()
        screen.debug()
    })
})