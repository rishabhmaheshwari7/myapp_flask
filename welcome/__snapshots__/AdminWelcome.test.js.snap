// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Admin Component Should render the admin welcome text correctly 1`] = `
Array [
  <div
    class="welcome-outer-container"
    role="welcome"
  >
    <img
      alt="Admin Welcome"
      class="welcome-image"
      src="https://res.infoq.com/articles/agile-team-maturity-assessment/en/smallimage/1G_yz7zg-1624490275821.jpg"
    />
    <div
      class="welcome-container"
    >
      <h2>
        Welcome to the Admin Dashboard
      </h2>
      <br />
      <h1>
        MAPLE
      </h1>
      <br />
      <br />
      <p>
        Maturity Assessment Platform : Lowe's Engineering
      </p>
    </div>
  </div>,
]
`;
